mono_repo/test/travis_command_test.dart

```diff
@@ -287,6 +287,107 @@ name: pkg_a
         ));
   });
 
+  test('`dart` key value is no-op with group', () async {
```

Comment:
test name doesn't seem to line up with the test? (it isn't a no-op, it is overriding the top level dart key)

Reply:
Since each group defines its Dart SDKs explicitly, the `dart: [no_op_dart_var]` is ignored. If you look at the output, there is no test ran on `dart: "no_op_dart_var"`

Reply:
In a later CL I handle this, allowing you to omit the top-level key and to print a warning if it's ignored

This helps if you think you've enabled `stable` by adding it to the the `dart` key, but you really haven't since all of the groups name their SDKs explicitly.

Reply:
Oh I read this as referring to the wrong `dart` key (the one for the individual job not the top level one). Maybe just `s/dart/top level dart`.

Reply:
Fixed!